packages/cli/src/zosfiles/create/Create.options.ts

```diff
@@ -36,6 +36,7 @@ export const ZosFilesCreateExtraOptions: { [key: string]: ICommandOptionDefiniti
      * @type {ICommandOptionDefinition}
      * @deprecated
      */
+    // eslint-disable-next-line deprecation/deprecation
```

Comment:
Not requesting to change anything, but it seems weird that `eslint-disable` comment is needed here, since we're just defining a deprecated variable not referencing it elsewhere. 🤔

Reply:
Yep, it is weird. But it really doesn't like that we are defining a deprecated option.